generator/testdata/rust/gclient/golden/model.rs

```diff
@@ -163,39 +163,43 @@ pub struct SecretVersion {
     ///  destroyed, the version is moved to disabled state and it is scheduled for
     ///  destruction. The version is destroyed only after the
     ///  `scheduled_destroy_time`.
-    pub scheduled_destroy_time: Option<String>,
+    pub scheduled_destroy_time: Option<String> /* TODO(#77) - handle .google.protobuf.Timestamp */,
 
     /// Output only. The customer-managed encryption status of the
     ///  [SecretVersion][google.cloud.secretmanager.v1.SecretVersion]. Only
     ///  populated if customer-managed encryption is used and
     ///  [Secret][google.cloud.secretmanager.v1.Secret] is a Regionalised Secret.
-    pub customer_managed_encryption: Option<CustomerManagedEncryptionStatus>,
+    pub customer_managed_encryption: Option<crate::CustomerManagedEncryptionStatus>,
 }
 
-#[derive(Clone, Debug, Default, Deserialize, Serialize)]
-pub struct SecretVersion_State(i32);
+/// Defines additional types related to SecretVersion
+pub mod secret_version {
 
-impl SecretVersion_State {
+    #[derive(Clone, Debug, Default, Deserialize, Serialize)]
+    pub struct State(i32);
 
-    // Not specified. This value is unused and invalid.
-    pub const SecretVersion_StateUnspecified: SecretVersion_State = SecretVersion_State(0);
+    impl State {
 
-    // The [SecretVersion][google.cloud.secretmanager.v1.SecretVersion] may be
-    // accessed.
-    pub const SecretVersion_Enabled: SecretVersion_State = SecretVersion_State(1);
+        // Not specified. This value is unused and invalid.
```

Comment:
Looks like we are missing an extra `/` in the template.

Reply:
Agree, created #84